app/frontend/entrypoints/application.scss

```diff
@@ -1,8 +1,7 @@
-$govuk-images-path: "@govuk/assets/images/";
-$govuk-fonts-path: "@govuk/assets/fonts/";
+$govuk-assets-path: "@govuk/assets/";
 $govuk-global-styles: true;
 
-@import "@govuk/all";
+@import "pkg:govuk-frontend";
 @import "dfe-autocomplete/src/dfe-autocomplete";
```

Comment:
Can we also use NodePackageImporter with dfe-autocomplete?

Reply:
Not as neatly (their package.json isn't currently configured to provide a Sass file, so we'll still have to configure the path to the file). I think it's worth doing anyway, for explicitness and consistency.

Reply:
Not as neatly (their package.json isn't currently configured to provide a Sass file, so we'll still have to configure the path to the file). I think it's worth doing anyway, for explicitness and consistency.

Reply:
Done!